RFC-008-release-versioning.md

```diff
@@ -0,0 +1,238 @@
+# RFC008 - Release Versioning Strategy
+
+## Proposed by
+
+Aaron Peters
+
+
+## Current status
+
+See the status indicated on the pull request description.
+
+
+## Proposal overview
+
+As The Good Docs Project moves towards a formal release structure, this RFC proposes a standard versioning scheme to apply at least to the project's Product Outputs, as defined by outputs directly employed by users in their own documentation efforts. For example, this includes the collection of templates (which users copy as a starting point for their own documents), but not whitepapers (which serve to inform their audience, but are not used verbatim).

```

Comment:
While there is less of a need to use versioning for whitepapers (and other documents), I can think of a few use cases where I might use semantic versioning for white papers. Especially if there are multiple editors/reviewers of the white paper. So maybe we should drop examples of where semantic versioning is not used, and leave it to personal choice. (If you really want a non-versioning article, I suggest referring to a news article.)
```suggestion
As The Good Docs Project moves towards a formal release structure, this RFC proposes a standard versioning scheme to apply at least to the project's Product Outputs, as defined by outputs directly employed by users in their own documentation efforts. For example, this includes the collection of templates (which users copy as a starting point for their own documents).
```

Reply:
I made a major mistake and accidentally merged this RFC in prematurely. I added your suggestion to the body of the RFC.